### ISSUE TITLE
Is there a bug in the code? For example, the training loss loss_a is not feedback

### ISSUE THREAD

**Issue by scevie** [hilbert9221/NRI-MPM#2]
Hello dear author:

I reproduced your code.

However, I found that the Loss_a in your Train is not feedback. Variable Loss_a in trian function in file XNRI.py (line 57).

**Comment by hilbert9221** [comment]
Thank you for trying my code! The local variable loss_a is defined to record the overall loss of all samples within an epoch. It is used for monitoring the training process other than back-propagation. The parameters of the model are updated within the function self.train_nri at Line 56. You may also check Line 111 in function self.train_nri.

Maybe you mean that it is not a good coding style. I would consider return loss_a.item(), a standard python number, to avoid being ambiguous.
